autogen/code_utils.py

```diff
@@ -247,15 +247,17 @@ def get_powershell_command():
 def _cmd(lang: str) -> str:
     if lang in PYTHON_VARIANTS:
         return "python"
-    if lang.startswith("python") or lang in ["bash", "sh"]:
+    elif lang.startswith("python") or lang in ["bash", "sh"]:
         return lang
-    if lang in ["shell"]:
+    elif lang in ["shell"]:
```

Comment:
Thanks a lot for your contribution!!! 💪 💪 💪 
Just want to ask you to avoid these changes, as when the `if` conditions are true the function returns something (it halts). Your changes in these lines just add noise to your PR.

Reply:
This happens again in lines 254 and 256.

Reply:
Thank you for your feedback ! I reverted the changes.